app/models/maintenance_stats/queries/github/commit_count_query.rb

```diff
@@ -0,0 +1,52 @@
+module MaintenanceStats
+  module Queries
+    module Github
+      class CommitCountQuery < BaseQuery
+        CommitCountsQuery = Rails.application.config.graphql.client.parse <<-'GRAPHQL'
+          query($owner: String!, $repo_name: String!, $start_date: GitTimestamp!) {
+            repository(owner: $owner, name: $repo_name) {
+              defaultBranchRef {
+                target {
+                  ... on Commit {
+                    history(since: $start_date) {
+                      totalCount
+                    }
+                  }
+                }
+                name
+              }
+            }
+          }
+        GRAPHQL
+
+        @@valid_params = [:owner, :name, :start_date]
```

Comment:
You probably don't want class variables here. Looks like these can be constants anyway?